build.sbt

```diff
@@ -1,46 +1,50 @@
-import play.sbt.routes.RoutesKeys
 import uk.gov.hmrc.DefaultBuildSettings
 
-ThisBuild / majorVersion := 0
-ThisBuild / scalaVersion := "2.13.12"
+ThisBuild / majorVersion := 1
+ThisBuild / scalaVersion := "3.3.5"
 
 lazy val appName: String = "find-your-national-insurance-number"
 
 lazy val scoverageSettings = {
   import scoverage.ScoverageKeys
   Seq(
-    // Semicolon-separated list of regexs matching classes to exclude
     ScoverageKeys.coverageExcludedPackages := "<empty>;Reverse.*;models/.data/..*;;models.*;.*(AuthService|BuildInfo|Routes).*",
-    ScoverageKeys.coverageMinimumStmtTotal := 80,
+    ScoverageKeys.coverageMinimumStmtTotal := 78,
     ScoverageKeys.coverageFailOnMinimum := true,
-    ScoverageKeys.coverageHighlighting := true,
-    scalacOptions += "-Wconf:cat=unused-imports&src=routes/.*:s"
+    ScoverageKeys.coverageHighlighting := true
   )
 }
 
+lazy val commonScalacOptions = Seq(
+  "-unchecked",
+  "-deprecation",
+  "-feature",
+  "-Wconf:msg=unused&src=routes/.*:s",
+  "-language:noAutoTupling",
+  "-Wvalue-discard",
+  "-Xfatal-warnings",
```

Comment:
Don't we want -Werror, -Wunused etc here?

Reply:
I added Werror, Wunused is enabled by default in sbt for all new versions, so will not be required. I missed adding Werror as normally we dont add that as a part of scala 3 upgrade, its already there in most of the repos, Guess this one is different.